server/prompt.ts

```diff
@@ -20,15 +20,17 @@ Avoid wrapping up the plot too quickly.
 Remember, your main goal is to tell a full and engaging story.
 If you rush through it, children might lose interest.
 Make the story as exciting and detailed as possible, always adding more rather than summarizing.
+
+If you are not able to write the story for whatever reason you should return this message: "ERROR: could not write the story"
```

Comment:
will this replace the messages that came up before about being innappropriate?or just when it cant connect to the gemini/etc? If it's including when it is not appropriate, maybe the message could say please review input choices or something

Reply:
changed it , an alert will show up when the prompt is inappropriate